dev-requirements.txt

```diff
@@ -1,4 +1,3 @@
--e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@master#egg=cloudify-dsl-parser==3.2a2
```

Comment:
dude.. you gotta stop changing the same things in different branches. You're gonna lose your mind and make mistakes merging these.